node_modules/col-activities/lib/api.js

```diff
@@ -35,6 +37,26 @@ var log = require('col-core/lib/logger')('col-activities');
 var UserConstants = require('col-users/lib/constants');
 
 var ActivitiesDefaults = require('./default');
+var DailyNotifications = require('./notifications/daily');
+
+/* Notifications */
+
+// When the emails should go out. For now, we're just using the server's timezone
+var hour = config.get('email.dailyHour');
+
+// Figure out the exact moment when to send out the emails
+var emailMoment = moment().minutes(0).seconds(0);
+var now = moment();
+if (now.hours() < hour) {
+  emailMoment = emailMoment.hours(hour)
+} else {
+  emailMoment = emailMoment.add(1, 'day').hours(hour);
+}
+
+var diff = emailMoment.diff(now);
+log.info('Scheduling daily notifications for %s (in %d ms)', emailMoment.format(), diff);
+setTimeout(DailyNotifications.collect, diff);
```

Comment:
I think the emails will only be sent once. I can't see anything that set the interval to send the emails out every day.